packages/manager/apps/telecom/src/app/telecom/pack/xdsl/access/comfortExchange/comfort-exchange.html

```diff
@@ -74,22 +74,42 @@
                     data-translate="xdsl_access_comfort_exchange_warning_part2"
                 ></p>
             </oui-message>
-            <div
-                class="mb-2"
-                data-translate="xdsl_access_comfort_exchange_confirm_title"
-            ></div>
-            <div
-                class="mb-2"
-                data-translate="xdsl_access_comfort_exchange_confirm_subtitle"
-            ></div>
+            <!-- CONTACT CHOICE -->
+            <div class="contact-choice col-xs-12 col-md-6">
+                <h3
+                    data-translate="xdsl_access_comfort_exchange_shipping_contact"
+                    class="mb-3"
+                ></h3>
 
-            <oui-button
-                data-variant="primary"
-                data-ng-disabled="$ctrl.isAvailable"
-                data-ng-click="$ctrl.comfortExchange()"
-            >
-                <span data-translate="xdsl_modem_exchange"></span>
-            </oui-button>
+                <ovh-contact
+                    data-ng-model="$ctrl.shipping"
+                    data-ovh-contact-choice-options="$ctrl.ovhContactOptions"
+                >
+                    <div class="text-center">
+                        <oui-spinner></oui-spinner>
```

Comment:
No conditions on the `oui-spinner` display ?

Reply:
Not for this spinner

Reply:
Not needed indeed. This spinner is use as transclude for using the `ovh-contact` component which has been made at a time where spinners were not the same between universes :-$